packages/pricing-features/components/sticky-tiers/index.tsx

```diff
@@ -36,7 +36,7 @@ export default function StickyTiers({
 
   return (
     <div
-      className={classNames(s.stickyTiers, isVisible && s.isVisible)}
+      className={classNames(s.wrapper, isVisible && s.isVisible)}
```

Comment:
This wrapper div was added to hide gradient overflow from the hero where it is implemented. The hero needs to have visible overflow to show the drop shadow effect on the pricing tiers section.